### PR TITLE
chore Update OTEL Operator to v0.53.0 release

### PR DESCRIPTION
**Pull request by jaronoff97**
This PR updates the OTEL operator the [latest v0.53.0 release](https://github.com/open-telemetry/opentelemetry-operator/pull/927)

closes https://github.com/open-telemetry/opentelemetry-helm-charts/issues/227

### CODE REVIEW

File: charts/opentelemetry-operator/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.7.0
+version: 0.7.1
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/Chart.yaml:3]:
```suggestion
version: 0.8.0
```

**Review by TylerHelmuth — COMMENTED** [review]
Was there a huge change in the CRD from 0.51.0 to 0.53.0?  Seems like a ton of changes.

**Comment by TylerHelmuth** [comment]
@abelperezok please review to ensure this will allow https://github.com/open-telemetry/opentelemetry-helm-charts/pull/218

**Comment by abelperezok** [comment]
@TylerHelmuth I don't see any conflicting change with #218 
Not sure about the details in CRDs changes, maybe @jaronoff97 can provide some ideas.

File: charts/opentelemetry-operator/Chart.yaml (hunk 1 above)
Comment (jaronoff97) [reply charts/opentelemetry-operator/Chart.yaml:3]: Most of the changes were white space, there was only one additive change.

**Comment by jaronoff97** [comment]
The CRD changes can be seen below:
Screen Shot 2022-06-15 at 8 12 41 PM

**Review by TylerHelmuth — APPROVED** [review]
@jaronoff97 thanks for taking care of this.  @dmitryax and I don't do much with the operator or the operator's chart so help is always appreciated.

**Pull request merged**